template.js

```diff
@@ -643,6 +643,8 @@ function addAppData(eventData, mappedData) {
   if (eventData.windows_attribution_id)
     mappedData.app_data.windows_attribution_id =
       eventData.windows_attribution_id;
+  if (eventData.referrer_url)
```

Comment:
Shouldn't this check be against page_referrer?

Reply:
it should